src/OpenTelemetry.Instrumentation.ElasticsearchClient/ElasticsearchClientInstrumentationOptions.cs

```diff
@@ -40,6 +40,11 @@ public class ElasticsearchClientInstrumentationOptions
         /// </summary>
         public bool SetDbStatementForRequest { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a max length allowed for the db.statement attribute. Default value: 4096.
+        /// </summary>
+        public int MaxDbStatementLength { get; set; } = 4096;
```

Comment:
Even though there hasn't been a stable release of this package yet, tt would be nice to have the publicAPI analyzers turned on for this project to detect any breaking changes.

Reply:
Not sure how to do that.

Reply:
You could refer to this doc for more information: https://github.com/dotnet/roslyn-analyzers#microsoftcodeanalysispublicapianalyzers

Reply:
@ejsmith I have created a PR #374 to add public API analyzer to ElasticSearchClient project. Please review it.